pterodactyl.php

```diff
@@ -279,7 +279,7 @@ public function editService(
         $service_helper = new PterodactylService();
 
         // Load user account
-        $pterodactyl_user = $this->apiRequest('Users', 'getByExternalId', [$service->client_id]);
+        $pterodactyl_user = $this->apiRequest('Users', 'getByExternalId', ['blesta-' . $service->client_id]);
```

Comment:
Need to update this to `"bl-" . $service->client_id`. This API request should also only be made if `$vars['use_module']` is true.

The return key/value pairs should also be updated to default the username to the currently-saved username on the service.